scripts/measures/get-inverse-measures-from-pdfs.js

```diff
@@ -0,0 +1,59 @@
+// running this will print out whether each measure is an inverse measure
+// claims pdfs can be downloaded from https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip
+// unzip and give the path to the Claims-Quality-Measures
```

Comment:
This should be both claims and registry quality measures, right? same for line below